docs/users/cli-usage.md

```diff
@@ -29,6 +29,109 @@ Available Commands:
+```
+Successfully started wordpress_site
+Your application can be reached at: http://wordpress_site.ddev.local
+```
+Quickstart instructions regarding database imports, can be found under [Database Imports](https://ddev.readthedocs.io/en/latest/users/cli-usage/#database-imports).
+### Drupal 7
+Beginning to use ddev with a Drupal 7 site is as simple as cloning the site's repository and checking out its directory.
+```
+git clone https://github.com/user/my_drupal7_site
+cd my_drupal7_site
+```
+Now to start working with ddev. Inside of your site's working directory, enter the following command:
+```
+ddev config
+```
+
+_Note: ddev config will prompt you for a site name and docroot._
+
+After you've run `ddev config` you're ready to start running your site. Run ddev using a simple:
+```
+ddev start
+``` 
+When running `ddev start` you should see output informing you that the site's environment is being started. If startup is successful, you'll see a message like the one below telling you where the site can be reached.
+```
+Successfully started my_drupal7_site
+Your application can be reached at: http://my_drupal7_site.ddev.local
+```
+Quickstart instructions regarding database imports, can be found under [Database Imports](https://ddev.readthedocs.io/en/latest/users/cli-usage/#database-imports).
+### Drupal 8
+Getting started with Drupal 8 sites in ddev is a friendly process that can be started by either cloning a git repository or using a new or existing composer project.
+
+**Git**
+```
+git clone https://github.com/user/my_drupal8_site
+cd my_drupal8_site
+```
+
+**Composer**
+```
+composer create-project drupal-composer/drupal-project:8.x-dev my_drupal8_site --stability dev --no-interaction
+cd my_drupal8_site
+```
+_You can find more information on composer and how to use it [here](https://github.com/drupal-composer/drupal-project)._
+
+The next step is to configure ddev. Inside of your site's working directory, enter the following command:
+```
+ddev config
+```
+
+_Note: ddev config will prompt you for a site name and docroot._
+
+After you've run `ddev config` you're ready to start running your site. Run ddev using a simple:
+```
+ddev start
+``` 
+When running `ddev start` you should see output informing you that the site's environment is being started. If startup is successful, you'll see a message like the one below telling you where the site can be reached.
+```
+Successfully started my_drupal8_site
+Your application can be reached at: http://my_drupal8_site.ddev.local
+```
+**Next:** Learn how to import a database in the [Database Imports](https://ddev.readthedocs.io/en/latest/users/cli-usage/#database-imports) section.
+#### Database Imports
+**Important:** Before importing any databases for your site, please remove its' wp-config.php if using Worpress - or settings.php file in the case of Drupal 7/8, if present. 
```

Comment:
We're in the wordpress section right? So no need to mention settings.php here? Where you mention both though, probably "if it exists". Back in the day, settings.php was never checked in. Recently it has been, and a settings.local.php is used for db config (you didn't used to be able to check in settings.php because it had sensitive config in it)

Reply:
This is a separate section that all three of the quickstarts (WP, D7, & D8) all use as a reference which is why that statement covers all of their scenarios. It doesn't make sense to dedicate a DB Imports section for each app type simply because the information literally gets repeated aside from the first line's disclaimer.